edgelet/edgelet-http-workload/src/lib.rs

```diff
@@ -126,14 +208,23 @@ where
             manifest_trust_bundle: "test-manifest-trust-bundle".to_string(),
             edge_ca_cert: "test-ca-cert".to_string(),
             edge_ca_key: "test-ca-key".to_string(),
+            edge_ca_auto_renew: None,
         };
 
+        // We won't use the renewal sender, but it must be created to construct the
+        // Service struct. Note that we drop the renewal receiver, which will cause
+        // tests to panic if they use the renewal sender.
+        let (renewal_tx, _) =
+            tokio::sync::mpsc::unbounded_channel::<edgelet_core::WatchdogAction>();
```

Comment:
I don't understand this.

- Since the Sender will never work, why create it? "it must be created to construct the Service struct" <- but that's because you added the field to the struct.

- `EdgeCaRenewal::write_credentials` *is* using this Sender to `send()` the WatchdogAction to restart the modules, though it only prints a warning when the send fails.

- If the send is a no-op, what's going to restart the modules?

Reply:
This function is test code. The real code does use the Sender, but not the tests.

Reply:
Ah, so it is.